src/features/search/redux/reducer.ts

```diff
@@ -14,4 +15,15 @@ export const search: Reducer<string, UpdateTickerAction> = (
             return state;
         }
     }
-};
\ No newline at end of file
+};
+
+export const stockList: Reducer<StockListItem[], UpdateStockListAction> = (state = [], action) => {
```

Comment:
Q: How come you've decided to separate the `stockList` state from the `search` state?

Reply:
So i decided to separate them as I had socket.on in my search component and not in with the rest of my middleware.

Placing it into redux let me remove this and get everything into one place.

I'll also be writing more into my middleware to allow for the removal of the query emit.

Reply:
Not sure I follow.
What I mean is, why not combine the `search` and `stockList` reducers?
Do you consider them to be 2 distinct entities, used by different parts of the app? Or are they intrinsically linked?
Former: agreed on the split.
Latter: I'd say combine.

Reply:
ah okay sorry I know what you mean now and yes I did think about that whilst writing it and they are linked. I'll combine into a single reducer and push.

Reply:
corrected and pushed